testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jca/metrics/data-sources/complex-driver.xml

```diff
@@ -6,7 +6,7 @@
 <subsystem xmlns="urn:jboss:domain:datasources:7.2">
     <datasources>
         <drivers>
-            <driver name="name" module="com.h2database.h2" major-version="2" minor-version="4">
+            <driver name="name" module="com.h2database.h2" major-version="2" minor-version="2">
```

Comment:
We'll probably have to use an expression for this and drive the system property based on what's being tested.

Unless the uses of this config aren't really deriving any value from setting this, in which case we can just remove this. AIUI these two version attributes are used to compare the config against what the driver itself reports, failing if there's a mismatch. I assume that's why the test fails in the WildFly Preview job. If an attribute isn't set it doesn't do the comparison. I don't see any other use.